### PR TITLE
Maven support

### PR DESCRIPTION
**Pull request by SiarheiKhmalEY**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+<!--        <dependency>-->
```
Comment (jaroslawwoloch) [inline pom.xml]: commented  -to remove

File: src/main/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    public List<String> readFromFile(Path path) {
+        logger.warn("Starting reading data");
+
+        List<String> stringList = new ArrayList<>();
+        try (BufferedReader bufferedReader = Files.newBufferedReader(Objects.requireNonNull(path))) {
+            while (bufferedReader.ready()) {
+                stringList.add(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            logger.error("Can not read file from path " + path);
```
Comment (jaroslawwoloch) [inline src/main/java/Main.java]: add exception to logger to log stack trace

**Review by jaroslawwoloch — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (SiarheiKhmalEY) [reply pom.xml]: Already done.

File: src/main/java/Main.java (hunk 2 above)
Comment (SiarheiKhmalEY) [reply src/main/java/Main.java]: Already done.

Comment (jaroslawwoloch) [reply src/main/java/Main.java]: first. only author of first comment accept fixes and mark resolve as resolved.
second. add exception to log mean add second parameter to logger method with throwable - logging system grab stacktrace autoamtically. - please correct this way
```suggestion
            logger.error("Can not read file from path " + path, e);
```

**Review by jaroslawwoloch — CHANGES_REQUESTED** [review]

File: src/main/java/Main.java (hunk 2 above)
Comment (SiarheiKhmalEY) [reply src/main/java/Main.java]: 1) Sorry for that.
2) Already done. Thank you for the hint.

Comment (jaroslawwoloch) [reply src/main/java/Main.java]: 1. it's unnecessary - this is a part of CodeReview and learning process
good job

**Review by jaroslawwoloch — APPROVED** [review]

**Pull request merged**